.github/workflows/release.yml

```diff
@@ -68,8 +68,8 @@ jobs:
           BRANCH: gh-pages
           FOLDER: build/docs
           DOC_FOLDER: gh-pages
-          COMMIT_EMAIL: behlp@objectcomputing.com
-          COMMIT_NAME: Puneet Behl
+          COMMIT_EMAIL: 'grails-build@users.noreply.github.com'
+          COMMIT_NAME: 'grails-build'
```

Comment:
We can dry it up by using `${{ env.GIT_USER_EMAIL }}` and `${{ env.GIT_USER_NAME }}` here.

Reply:
I don't know, I have changed so many to grails-build, at this point.  I lean slightly towards grails-build over ${{ env.GIT_USER_NAME }}

Reply:
Ok, it's hopefully not going to change anytime soon again. But to me, it feels wrong to duplicate values. I believe the environment vars are also used in the `micronaut-projects/github-actions/post-release@master` action.

Reply:
Is ${{ env.GIT_USER_EMAIL }}='grails-build@users.noreply.github.com' and ${{ env.GIT_USER_NAME }}='grails-build' instead of currently logged in user?  I may have misunderstood.

Reply:
Yes 👍, they are set higher up by in the file.

Reply:
https://github.com/grails/grails-database-migration/pull/376